packages/shared/lib/core/router/subrouters/profile-setup-router.ts

```diff
@@ -39,13 +39,8 @@ export class ProfileSetupRouter extends Subrouter<ProfileSetupRoute> {
                 const _profileRecoveryType = params?.profileRecoveryType
                 if (_profileRecoveryType) {
                     if (_profileRecoveryType === ProfileRecoveryType.Mnemonic) {
-                        nextRoute = ProfileSetupRoute.EnterName
                         this.parentRouter.next()
-                    } else if (
-                        _profileRecoveryType === ProfileRecoveryType.Stronghold ||
-                        _profileRecoveryType === ProfileRecoveryType.File
-                    ) {
```

Comment:
What happens if the recovery type is a `File`? Also there is duplication for `this.parentRouter.next()`